benchmarks/buiter_et_al_2016_jsg/exp_2_low_resolution.prm

```diff
@@ -0,0 +1,199 @@
+set Use years in output instead of seconds = false
+set CFL number				   = 0.5
+set Maximum time step			   = 1e5 #3.6
+
+set Nonlinear solver scheme                = single Advection, iterated Stokes
+set Nonlinear solver tolerance             = 1e-7
+set Max nonlinear iterations               = 100
+
+subsection Solver parameters
+  subsection Stokes solver parameters
+    set Linear solver tolerance = 1e-8
+    set Number of cheap Stokes solver steps = 0
+  end
+end
+
+set Output directory                       = Brittle_thrust_wedge_output
+set Pressure normalization                 = no
+set Timing output frequency                = 20
+set Resume computation                     = auto
+
+### Parameters describing the model
+
+# Model geometry (36 x 8 cm, the resolution is 2 mm/cell)
+ subsection Geometry model
+   set Model name = box
+ 
+   subsection Box
+     set X repetitions = 180
+     set Y repetitions = 40
+     set X extent      = 0.36
+     set Y extent      = 0.08
+   end
+ end
+
+# Mesh refinement specifications 
+subsection Mesh refinement
+  set Initial adaptive refinement        = 0
+  set Initial global refinement          = 0
+  set Time steps between mesh refinement = 0
+end
+
+# Boundary conditions
+subsection Boundary composition model
+  set Fixed composition boundary indicators   = bottom, left, right
+end
+
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators   = bottom, top
+end
+
+subsection Boundary velocity model
+  set Zero velocity boundary indicators = bottom
+  set Tangential velocity boundary indicators = left
+  set Prescribed velocity boundary indicators = right:function
+    subsection Function
+      set Variable names      = x,y
+      set Function constants  = cm=0.01, h=3600.
+      set Function expression = if(y>0.002, -2.5*cm/h, -(y/0.002)*2.5*cm/h); 0
+   end
+end
+
+subsection Boundary traction model
+  set Prescribed traction boundary indicators = top: zero traction
+end
+
+# Number and names of compositional fields
+subsection Compositional fields
+  set Number of fields = 6
+  set Names of fields = qsand,csand,bound,block,sticky_air,total_strain
+end
+
+
+# Spatial domain of different compositional fields
+subsection Initial composition model
+  set Model name = function
+  subsection Function
+    set Variable names      = x,y
+    set Function expression = if((x<=0.35 & y<=0.012 & y>0.002) || (x<=0.35 & y>0.022 & y<=0.032), 1,0); \
+                              if((x<=0.35 & y>0.012 & y<=0.022), 1,0); \
+                              if((x>0.35 & x<=0.352 & y>0.002) ||(y<=0.002), 1, 0); \
+                              if(x>0.352 & y>0.002,1,0); \
+			      if(x<=0.35 & y>0.032,1,0); 0
+  end
+end
+
+# Composition boundary conditions
+subsection Boundary composition model
+  set List of model names = initial composition
+end
+
+# Temperature boundary conditions
+subsection Boundary temperature model
+  set List of model names = box
```

Comment:
You can change this to `set List of model names = initial temperature` and then remove the `Box` subsection below.